Project1.ts

```diff
@@ -0,0 +1,80 @@
+const axios = require('axios');
+const { MongoClient } = require('mongodb');
+const express = require('express');
+const retry = require('async-retry');
+
+const uri = "mongodb://127.0.0.1:27017/";
+const client = new MongoClient(uri);
+const app = express();
+const port = 3000;
+const mainUrl = "https://jsonplaceholder.typicode.com/users/"
+
+
+app.get('/users/', async (req, res) => {
+    await retry(async() => {
+
+    await client.connect();
+    const db = client.db("local");
+    const collection1 = db.collection("users");
+    var response = await axios.get(mainUrl);
+    let dataArray = response.data;
+    let dataPopArray =  dataArray.map(personData => {
+        let tempNameArray =  personData.name.split(' ');
+        if (tempNameArray[0] == 'Mr.' || tempNameArray[0] == 'Mrs.') {
```

Comment:
I don't understand the logic, but regarding to the code:
Use the strict equality operator === instead of the loose equality operator